src/core/Util.js

```diff
@@ -156,8 +156,8 @@ export function template(str, data) {
 // mobile devices (by setting image `src` to this string).
 export const emptyImageUrl = 'data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=';
 
-const requestFn = window.requestAnimationFrame;
-const cancelFn = window.cancelAnimationFrame;
+const requestFn = typeof window === 'undefined' ? () => { } : window.requestAnimationFrame;

```

Comment:
I think `L.Util.falseFn` would be better. https://leafletjs.com/reference.html#util-falsefn

Reply:
done 👍🏻